src/value.rs

```diff
@@ -540,6 +545,16 @@ impl Value {
     .unwrap()
   }
 
+  pub fn to_external<'s>(
+    &self,
+    scope: &mut HandleScope<'s, ()>,
+  ) -> Local<'s, External> {
+    unsafe {
+      scope.cast_local(|sd| v8__Value__ToExternal(self, sd.get_isolate_ptr()))
+    }
+    .unwrap()
+  }
```

Comment:
I can see why you put this here but, contrary to the other methods, `v8::External::Cast()` is distinctly unsafe - it's basically a `reinterpret_cast` or `std::mem::transmute()`.

A better way to deal with converting from `Value` to `External` is through `.try_into()` (which might already work, not sure.)

Reply:
You're right. That's kind of overdesign and unnecessary. Removed.